src/cards/LockCard.js

```diff
@@ -0,0 +1,39 @@
+import {AbstractCard} from "./AbstractCard";
+
+/**
+ * Lock Card Class
+ *
+ * Used to create a card for controlling an entity of the lock domain.
+ *
+ * @class
+ * @extends AbstractCard
+ */
+class LockCard extends AbstractCard {
+  /**
+   * Default options of the card.
+   *
+   * @type {lockCardOptions}
```

Comment:
`lockCardOptions` should be added to file `src\cards\typedefs.js`

Reply:
not sure what to add because there aren't any options

Reply:
```js
/**
 * @typedef {abstractOptions} LockCardOptions Lock Card options.
 * @memberOf typedefs.cards
 */
```